install-tpl.sh

```diff
@@ -95,6 +95,9 @@ KOKKOS=${KOKKOS:-NO}
 KOKKOS=`check_valid KOKKOS`
 
 H5VERSION=${H5VERSION:-V110}
+FAODEL=${FAODEL:-OFF}
```

Comment:
I think these should be NO instead of OFF

Reply:
Fixed in commit f54a315.